views/js/pciCreator/ims/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -832,7 +921,7 @@ define([
 
                 this.$container = $(dom);
                 this.$toolbar = this.$container.find('.toolbar');
-                this.$input = this.$container.find('.math-entry-input');
+                this.inputs = [this.$container.find('.math-entry-input')];
```

Comment:
**thought:** Looking at the way the alternative answers are managed, I wonder if is the correct approach to rely on an array. The answers are indexed by their position in the array, leading to possible collisions and misses when elements are removed. Unless another means is used to identify the answers, I think this is error-prone, and it would need to be changed.

**suggestion:** Use a `Map` with unique identifiers for each answer. You can generate identifiers unique for the current session only, using an internal counter.
```suggestion
                const id = uid();
                this.inputs = new Map([id, this.$container.find('.math-entry-input')]);
```

```javascript
    // module wide
    let uidCounter = 0;
    const uid = () => `answer-${uidCounter++}`;
```